install/installer/scripts/kots-install.sh

```diff
@@ -0,0 +1,345 @@
+#!/bin/sh
+# Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+# Licensed under the MIT License. See License-MIT.txt in the project root for license information.
+
+# shellcheck disable=SC2050,SC2153
+
+set -e
+
+echo "Gitpod: Killing any in-progress installations"
+
+kubectl delete jobs.batch -n "${NAMESPACE}" -l component="gitpod-installer,cursor!=${CURSOR}" --force --grace-period 0 || true
+kubectl delete pod -n "${NAMESPACE}" -l component="gitpod-installer,cursor!=${CURSOR}" --force --grace-period 0 || true
+
+if [ "$(helm status -n "${NAMESPACE}" gitpod -o json | jq '.info.status == "deployed"')" = "false" ];
+then
+    echo "Gitpod: Deployment in-progress - clearing"
+
+    VERSION="$(helm status -n "${NAMESPACE}" gitpod -o json | jq '.version')"
+    if [ "${VERSION}" -le 1 ];
+    then
+        echo "Gitpod: Uninstall application"
+        helm uninstall -n "${NAMESPACE}" gitpod --wait || true
+    else
+        echo "Gitpod: Rolling back application"
+        helm rollback -n "${NAMESPACE}" gitpod --wait || true
+    fi
+fi
+
+echo "Gitpod: Create a Helm template directory"
+rm -Rf "${GITPOD_OBJECTS}"
```

Comment:
nit: Maybe document somewhere in the file about GITPOD_OBJECTS? :thinking: While its straightforward with other env's as they are input it could be confusing on what `kots-install.sh` emits.

Reply:
If it's ok with you, I won't fix in here - this won't be sticking around long as I've already started the next refactor and this probably won't survive. If it does, I'll fix it then

Reply:
Sounds good!